articles/quickstart/backend/java-spring-security/01-authorization.md

```diff
@@ -152,97 +136,37 @@ import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.ResponseBody;
+import org.json.JSONObject;
 
 @Controller
 @Component
-public class PhotosController {
-
-    @RequestMapping(value = "/login")
-    @ResponseBody
-    public String login() {
-        return "All good. You DO NOT need to be authenticated to call /login";
-    }
+public class APIController {
 
-    @RequestMapping(value = "/photos", method = RequestMethod.GET)
+    @RequestMapping(value = "/api/public", method = RequestMethod.GET, produces = "application/json")
     @ResponseBody
-    public String getPhotos() {
-        return "All good. You can see this because you are Authenticated with a Token granted the 'read:photos' scope";
+    public String publicEndpoint() {
+        return new JSONObject()
+                .put("message", "All good. You DO NOT need to be authenticated to call /api/public.")
+                .toString();
     }
 
-    @RequestMapping(value = "/photos", method = RequestMethod.POST)
+    @RequestMapping(value = "/api/private", method = RequestMethod.GET, produces = "application/json")
     @ResponseBody
-    public String createPhotos() {
-        return "All good. You can see this because you are Authenticated with a Token granted the 'create:photos' scope";
+    public String privateEndpoint() {
+        return new JSONObject()
+                .put("message", "All good. You can see this because you are Authenticated.")
+                .toString();
     }
 
-    @RequestMapping(value = "/photos", method = RequestMethod.PUT)
+    @RequestMapping(value = "/api/private-scoped", method = RequestMethod.GET, produces = "application/json")
     @ResponseBody
-    public String updatePhotos() {
-        return "All good. You can see this because you are Authenticated with a Token granted the 'update:photos' scope";
-    }
-
-    @RequestMapping(value = "/photos", method = RequestMethod.DELETE)
-    @ResponseBody
-    public String deletePhotos() {
-        return "All good. You can see this because you are Authenticated with a Token granted the 'delete:photos' scope";
-    }
-
-    @RequestMapping(value = "/**")
-    @ResponseBody
-    public String anyRequest() {
-        return "All good. You can see this because you are Authenticated.";
+    public String privateScopedEndpoint() {
+        return new JSONObject()
+                .put("message", "All good. You can see this because you are Authenticated with a Token granted the 'read:messages' scope")
+                .toString();
     }
 
 }
 ```
 
 To build and run the seed project, use the command: `mvn spring-boot:run`.
-
```

Comment:
Where is all this info that it's being removed going to be located now?

Reply:
The info about using the API will be in `02-using.md`